packages/back-end/test/permissions.test.ts

```diff
@@ -1807,3 +1811,636 @@ describe("hasReadAccess filter", () => {
     ]);
   });
 });
+
+describe("PermissionsUtilClass.canCreateMetric check", () => {
```

Comment:
Given that `canCreateMetric` is a wrapper around `checkProjectFilterPermission`, it feels like what we actually want to test is `checkProjectFilterPermission`. But, given `checkProjectFilterPermission` is a private method, I'm not aware of a pattern that would allow us to do so without having to mock `checkProjectFilterPermission`.

Open to suggestions.